todo-mvc/src/main.ts

```diff
@@ -89,12 +88,18 @@ app.registerStore('todo-store', todoStore);
 
 Object.keys(storeTodoActions).forEach((actionName) => {
 	const action: AnyAction = (<any> storeTodoActions)[actionName];
-	app.registerAction(actionName, action);
+	action.configure(app.registryProvider);
 });
 
 Object.keys(uiTodoActions).forEach((actionName) => {
 	const action: AnyAction = (<any> uiTodoActions)[actionName];
-	app.registerAction(actionName, action);
+	if (actionName === 'filter') {
```

Comment:
Do you think we should extract these to a separate action class? This logic seems like it could be brittle.